### ISSUE TITLE
[JOSS] Installation instructions

### ISSUE THREAD

**Issue by mathieuboudreau** [ritagnunes/PulseqDiffusion#4]
Hi,

The installation instructions on the README are somewhate lacking for me to be able to get up and running to use the software.  https://github.com/ritagnunes/PulseqDiffusion/blob/master/README.md

* I have MATLAB installed.
* I created a venv with python 3.6. I downloaded this github repository (even though those aren't in the installation instructions). I installed `pip install pypulseq` in it after activating the venv.
* I installed the NIfTI reader you recommended in Ref 7 of your README and placed it in a folder called 'nifti_tools'.
* I have FSL installed.

Now what am I supposed to do? Is it installed correctly now that I've done those steps?

**Comment by ritagnunes** [comment]
Yes, hopefully the Usage instructions that have been added to README.md in a recent commit will help to make this clearer.

**Comment by grlee77** [comment]
Yes, it seems this is not a traditional software library with an installation process, but is instead a series of Python and Matlab scripts. 

The Python scripts generate the sequence files that would be used on the scanner and the Matlab scripts are for reconstruction, generating gradient tables, etc.

The Python code can be run as long as the user has installed `pypulseq` as indicated in the README. Then there is no actual installation step for PulseqDiff itself, one just navigates to the folder with the scripts and can run them directly from the shell via
`python write_se_dwi_epi_bValue.py`, etc.

**Comment by ritagnunes** [comment]
At present we are using Matlab for reconstructing the images as this code spans from previous work, developed over time.
In any case we would like to point out that as these scripts don't rely on specific toolboxes, users who do not have access to Matlab should be able to use Octave instead. We are planning to provide a Python version of the reconstruction code in a future release.
